app/Http/Controllers/CardDAV/MonicaCardDAVBackend.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+namespace App\Http\Controllers\CardDAV;
+
+use Sabre\DAV, Sabre\CalDAV, Sabre\DAVACL, Sabre\VObject;
+use Auth;
+use Log;
+
+class MonicaCardDAVBackend implements \Sabre\CardDAV\Backend\BackendInterface {
+
+    /**
+     * Returns the list of addressbooks for a specific user.
+     *
+     * Every addressbook should have the following properties:
+     *   id - an arbitrary unique id
```

Comment:
Could we use our HashID here?

Reply:
I will try, need to see if we can use non-numeric values.